mkdocs.yml

```diff
@@ -1,53 +1,58 @@
-site_name: Gateway Integrations Guide
+site_name: Gateway Technical Guides
 theme:
-    name: material
-    #custom_dir: docs/overrides
-    palette:
-        scheme: hdruk
-    logo: images/logo.png
-    favicon: images/favicon.png
-    features:
-        - navigation.footer
-        - navigation.instant
-        - navigation.expand
-        - content.code.copy
-        - content.code.annotate
-        - toc.follow
+      name: material
+      #custom_dir: docs/overrides
+      palette:
+            scheme: hdruk
+      logo: images/logo.png
+      favicon: images/favicon.png
+      features:
+            - navigation.footer
+            - navigation.instant
+            - navigation.expand
+            - content.code.copy
+            - content.code.annotate
+            - toc.follow
 extra_css:
-    - stylesheets/custom.css
+      - stylesheets/custom.css
 
 nav:
-    - Home: index.md
-    - Gateway 2.0:
-          - Login: fe-login.md
-          - API Management: fe-creating-an-app.md
-          - FMA Setup: fe-creating-fma.md
-    - Metadata:
-          - Create: creating-metadata.md
-          - Validate: metadata-validation.md
-          - Exemplars: example-metadata.md
-    - Using the API:
-          - Datasets:
-                - Create: creating-a-dataset.md
-                - Retrieve: retrieving-a-dataset.md
-                - Update: updating-a-dataset.md
-                - Delete: deleting-a-dataset.md
-    - Using FMA:
-          - Test Server Example: fma-test-server.md
+      - Home: index.md
+      - Accessing Our APIs:
+            - What is a Private App: private-app-what.md
+            - Create a Private App: private-app-create.md
+            - Manage a Private App: private-app-manage.md
+      - Using the Gateway:
+            - Login: fe-login.md
+            - FMA Setup: fe-creating-fma.md
+      - Integrations:
+            - HDR UK Datasets Integration: integrations-hdruk-metadata.md
+      - Metadata:
```

Comment:
Is 'Metadata' here specifically for dataset metadata?

Reply:
Have adjusted, this was named incorrectly. fixed in next merge